prow/daily-release-qualification.sh

```diff
@@ -63,5 +63,18 @@ cp -R ${DAILY_BUILD}/install/* install/
 ARTIFACTS_DIR="${GOPATH}/src/github.com/istio-releases/daily-release/_artifacts"
 
 echo 'Running E2E Tests'
-ARGS="--test_logs_path="${ARTIFACTS_DIR}" --istioctl_url "${ISTIOCTL_STAGE_URL}" "$@""
-make e2e_all E2E_ARGS="${ARGS}"
+ARGS=( --test_logs_path="${ARTIFACTS_DIR}" --istioctl_url "${ISTIOCTL_STAGE_URL}" "$@" )
+
+EXTRA_E2E_ARGS=()
+EXTRA_E2E_ARGS+=( --mixer_tag="${TAG}" )
+EXTRA_E2E_ARGS+=( --pilot_tag="${TAG}" )
+EXTRA_E2E_ARGS+=( --proxy_tag="${TAG}")
```

Comment:
nit: add a white space before closing parenthesis to be consistent in style

Reply:
Done